### DISCUSSION TITLE
How much vulnerability information does trivy db have?

### DISCUSSION THREAD

**Discussion by hyeyoung-leee** [aquasecurity/trivy#3851] [Q&A] [answered]
Hi,
How much vulnerability information does trivy db have?

Is it the 197,462 shown in https://www.cve.org/?

**Comment by itaysk** [comment, marked as answer]
Trivy's vulnerability DB sources are listed here: https://aquasecurity.github.io/trivy/v0.38/docs/vulnerability/detection/data-source/
